homework.js

```diff
@@ -0,0 +1,61 @@
+//question 1 
+const summ = function(n) {
+    var sum = 0;
+    for (let i = 1; i<=n;i++)
+    sum += i ;
+    console.log(sum);
+    }
+    summ(5)
+VM562:5 15
+
+// question 2 
+let numbers = [ 2,4,6,8 ];
+
+let sum = numbers.reduce((a, b) => {
+    return a + b;
+});
```

Comment:
Question 2 specifies that you should make a function that gets the sum of all the **_even_** numbers in a group from 1 to the `n` number. 
Your solution will add all numbers not just even numbers.